playbooks/roles/tower/tasks/bootstrap.yml

```diff
@@ -59,3 +59,9 @@
 
 # Import Job Template bootstrapping
 - include_tasks: bootstrap_jobs.yml
+
+- name: Expose tmp directory to support job isolation
+  tower_settings:
+    name: "AWX_PROOT_SHOW_PATHS"
+    value:
+      - "/tmp"
```

Comment:
@pmccarthy can [this](https://github.com/pmccarthy/ansible-tower-configuration/tree/INTLY-3623_migrate_extra_vars_files_to_rhmi_installer#32-tower-configuration) step in the README be removed now that it is handled by this task?

Reply:
Yep, I'll update that too as part of this PR